### PR TITLE
Add bridge alert examples

### PR DESCRIPTION
**Pull request by stanlyDoge**
Signed-off-by: Stanislav Knot <sknot@redhat.com>

### Type of change
- Enhancement / new feature

### Description
Add bridge alert examples described in https://github.com/strimzi/strimzi-kafka-operator/issues/3327

Fixes https://github.com/strimzi/strimzi-kafka-operator/issues/3327
### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: examples/metrics/prometheus-install/prometheus-rules.yaml [hunk 1]
```diff
@@ -185,6 +185,46 @@ spec:
...
+        summary: 'Kafka Bridge consumer average fetch latency'
+        description: 'The average consumer commit latency is {{ $value }} on {{ $labels.clientId }}'
+    - alert: AvgConsumerCommitLatency
+      expr: strimzi_bridge_kafka_consumer_commit_latency_avg > 10
+      for: 10s
+      labels:
+        severity: warning
+      annotations:
+        summary: 'Kafka Bridge consumer average commit latency'
+        description: 'The average consumer commit latency is {{ $value }} on {{ $labels.clientId }}'
```
Comment (scholzj) [inline examples/metrics/prometheus-install/prometheus-rules.yaml:211]: Out of curiosity, how did you come up with the value of 10 (ms I guess?)?

Comment (ppatierno) [reply examples/metrics/prometheus-install/prometheus-rules.yaml:211]: As far as I can see they are "seconds" and 10 seconds of latency sound to be too high for an alarm.

Comment (stanlyDoge) [reply examples/metrics/prometheus-install/prometheus-rules.yaml:211]: It should be miliseconds. So if we have latency > 10ms for 10s, fire an alert.

Comment (scholzj) [reply examples/metrics/prometheus-install/prometheus-rules.yaml:211]: So have you done any tests to see that the 10ms latency is really something to alert about?

Comment (stanlyDoge) [reply examples/metrics/prometheus-install/prometheus-rules.yaml:211]: No, I do not know how to simulate latency

Comment (scholzj) [reply examples/metrics/prometheus-install/prometheus-rules.yaml:211]: I guess you should at lest try some performance test and watch what will be the average latency during it to set the alert right.

**Review by ppatierno — APPROVED** [review]
LGTM assuming the new numbers come from more tests on Kubernetes.

**Review by scholzj — APPROVED** [review]
I guess if Paolo is fine with it, who am I to disagree :-o

**Comment by scholzj** [comment]
@stanlyDoge I guess this could be merged now, right?

**Pull request merged**